setup.py

```diff
@@ -475,10 +479,10 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
 devel_hadoop = devel_minreq + hdfs + hive + kerberos + presto + webhdfs
 
 devel_all = (all_dbs + atlas + aws + azure + celery + cgroups + dask + datadog + devel + doc + docker +
-             elasticsearch + exasol + facebook + gcp + grpc + hashicorp + jdbc + jenkins + kerberos +
+             elasticsearch + exasol + facebook + google + grpc + hashicorp + jdbc + jenkins + kerberos +
```

Comment:
Can you create a providers variable that will only contain dependencies for providers? It's getting harder to review this list.
```
providers = (
  was + azure + google....
)
devel_all = (all + dbs + devel + doc + providers)

Reply:
Good Idea. I already have the list but I will bring it into setup.py

Reply:
Fixed.